Missionframework/stringtable.xml

```diff
@@ -949,11 +949,19 @@
             <German>KP Kistenfüller</German>
             <Polish>KP Cratefiller</Polish>
         </Key>
+        <Key ID="STR_KPLIB_DIALOG_CRATEFILLER_TITLETOOLS">
+            <English>KP Cratefiller Tools</English>
+            <German>KP Kistenfüller Werkzeuge</German>
```

Comment:
This sounds very awkward. It is a small "sidebar" in the dialog, right? Maybe remove the "KP Cratefiller" phrase as the main dialog window already shows that. And maybe there is a more descriptive/sensible word for the "side bar" with the player selection etc. which is more sensible than "Tools"

Reply:
Maybe something like `Equipment overview`?

Reply:
Yeah, sounds fine.